### PR TITLE
DOCSP-21728 Update Blue Banner EOL Alert for CM/OM pages

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [JIRA](https://jira.mongodb.org/browse/DOCSP-21728)

### CODE REVIEW

**Review by corryroot — APPROVED** [review]
LGTM!

**Review by schmalliso — APPROVED** [review]
Looks good, with the small reminder that you will need to set theme_backup_monitoring_eol to true in the config for the relevant branch(es) in the mms-docs repo.

**Review by schmalliso — APPROVED** [review]

File: themes/mms-onprem/page.html [hunk 1]
```diff
@@ -50,6 +50,22 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
...
   {%- endif %}
 
+  {%- if theme_backup_monitoring_eol %}
+    <div class="alert alert-info alert-dismissible fade in" role="alert" style="margin-left: 20px">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Ops 
+          Manager will no longer support Backup and Monitoring for
```
Comment (jwilliams-mongo) [inline themes/mms-onprem/page.html:60]: Q: Is September 30 the release date for the version(s) of Ops Manager that drop this support? It's not clear to me if this change will affect all current OM versions, only 6.0, or only ones released after Sep 30.

**Review by jwilliams-mongo — APPROVED** [review]
@kanchana-mongodb LGTM with one question that's probably best suited for product

File: themes/mms-onprem/page.html (hunk 1 above)
Comment (kanchana-mongodb) [reply themes/mms-onprem/page.html:60]: Asked in the [DOCSP ticket](https://jira.mongodb.org/browse/DOCSP-21728). Will await response before backporting.

File: themes/mms-cloud/page.html [hunk 2]
```diff
@@ -59,6 +59,21 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
...
   {%- endif %}
 
+  {%- if theme_backup_monitoring_eol %}
+    <div class="alert alert-info alert-dismissible fade in" role="alert" style="margin-left: 20px">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Cloud 
+          Manager will no longer support Backup and Monitoring for
```
Comment (atsansone) [inline themes/mms-cloud/page.html:69]:
```suggestion
          Manager will not support Backup and Monitoring for 
```

File: themes/mms-onprem/page.html (hunk 1 above)
Comment (atsansone) [inline themes/mms-onprem/page.html:60]: **nit:** Simplify the language.
```suggestion
          Manager will not support Backup and Monitoring for 
```

File: themes/mms-cloud/page.html [hunk 3]
```diff
@@ -59,6 +59,21 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
...
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Cloud 
+          Manager will no longer support Backup and Monitoring for 
+          MongoDB 2.6, 3.0, 3.2, or 3.4 after September 30, 2022.
+          Please <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">upgrade your MongoDB deployment</a>
+          or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate to Atlas.</a>
+        </h4>
```
Comment (atsansone) [inline themes/mms-cloud/page.html:73]: **issue:** Simplify the language: shorten the headline, be direct in the change, be explicit in the call to action without pleading.
```suggestion
        <h4 class="alert-heading" style="margin-bottom: 10px;">Backup, Monitoring Support on MongoDB 3.4 and Earlier Ends September 30, 2022</h4>
         Cloud Manager ends support Backup and Monitoring for MongoDB 2.6, 3.0, 3.2, or 3.4 on Friday, September 30, 2022.
         <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">Upgrade your MongoDB databases to 3.6 or later</a>
         or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate your databases to MongoDB Atlas.</a>

```

File: themes/mms-onprem/page.html [hunk 4]
```diff
@@ -50,6 +50,22 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">The MongoDB Agent has bee
...
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">Ops 
+          Manager will not support Backup and Monitoring for 
+          MongoDB 2.6, 3.0, 3.2, or 3.4 after September 30, 2022.
+          Please <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">upgrade your MongoDB deployment</a>
+          or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate to Atlas.</a>
+        </h4>
```
Comment (atsansone) [inline themes/mms-onprem/page.html:64]: **issue:** Simplify the language: shorten the headline, be direct in the change, be explicit in the call to action without pleading.
```suggestion
        <h4 class="alert-heading" style="margin-bottom: 10px;">Backup, Monitoring Support on MongoDB 3.4 and Earlier Ends September 30, 2022</h4>
         OpsManager ends support Backup and Monitoring for MongoDB 2.6, 3.0, 3.2, or 3.4 on Friday, September 30, 2022.
         <a href="{{theme_base_url}}/cloud-manager/tutorial/change-mongodb-version/">Upgrade your MongoDB databases to 3.6 or later</a>
         or <a href="{{theme_base_url}}/atlas/migration-to-atlas-choosing/">migrate your databases to MongoDB Atlas.</a>

```

**Review by atsansone — APPROVED** [review]

**Pull request merged**